Cargo.toml

```diff
@@ -30,3 +30,7 @@ stdweb = { version = "0.4.18", optional = true }
 
 [features]
 std = []
+# enables dummy implementation for wasm32-unknown-unknown
```

Comment:
Why have `wasm_dummy` and `dummy`. It seems like it would be much better to just have a single `dummy` feature.

Reply:
`wasm_dummy` is mainly intended for the `std` inclusion PR. We want use a constant seed only on `wasm32-unknown-unknown` and get a compilation error for all other unsupported targets. Since we can't enable features based on targets, I think using a separate `wasm_dummy` feature will be the most straightforward approach.

Reply:
Ok, so part of my confusion was that I assumed that the implementation for `std` integration would look like:

```rust
fn get_keys() -> [u64; 2] {
    let mut buf = [0u8; 16];

    #[cfg(not(target = "wasm32-unknown-unknown"))]
    getrandom::getrandom(&mut buf).expect("getrandom failure");

    let n = u128::from_ne_bytes(buf);
    [n as u64, (n >> 64) as u64]
}
```

In which case, we wouldn't need the `wasm_dummy` feature, as we wouldn't even compile `getrandom` on `wasm32-unknown-unknown`. We would just have the following in the `Cargo.toml`:
```toml
[target.'cfg(not(target = "wasm32-unknown-unknown"))'.dependencies]
getrandom = { version = "0.1.8", features = ['rustc-dep-of-std', 'dummy'] }
```

Reply:
Unfortunately `target = ".."` does not work in `cfg` expressions, see: rust-lang/rust#63217. I guess we could write something like `cfg(not(all(target_arch="wasm32", target_os="unknown", target_env="unknown")))`. It's a bit unwieldy, but it indeed allows us to remove the `wasm_dummy` feature.